<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDialog</class>
 <widget class="QDialog" name="ControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="imageFolderLabel">
       <property name="text">
        <string>&amp;Image Folder:</string>
       </property>
       <property name="buddy">
        <cstring>imageFolderPathEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="playModeLabel">
       <property name="text">
        <string>Play-Mode:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="linearModeButton">
         <property name="text">
          <string>Li&amp;near</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shuffleModeButton">
         <property name="text">
          <string>&amp;Shuffle</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="randomModeButton">
         <property name="text">
          <string>&amp;Random</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="loopLabel">
       <property name="text">
        <string>&amp;Loop:</string>
       </property>
       <property name="buddy">
        <cstring>loopCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="loopCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="displayTimeLabel">
       <property name="text">
        <string>Display-&amp;Time:</string>
       </property>
       <property name="buddy">
        <cstring>displayTimeDoubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="displayTimeDoubleSpinBox">
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="suffix">
        <string> sec</string>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>120.000000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="movieMinimumLoopCountLabel">
       <property name="text">
        <string>Movie Minimum Loop &amp;Count:</string>
       </property>
       <property name="buddy">
        <cstring>movieMinimumLoopCountSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="movieMinimumLoopCountSpinBox">
       <property name="specialValueText">
        <string>None</string>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="filterFiletypesLabel">
       <property name="text">
        <string>&amp;Filter Filetypes:</string>
       </property>
       <property name="buddy">
        <cstring>filterFiletypesCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="filterFiletypesCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPathEdit" name="imageFolderPathEdit">
       <property name="pathMode">
        <enum>QPathEdit::ExistingFolder</enum>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="allowEmptyPath">
        <bool>false</bool>
       </property>
       <property name="placeholder">
        <string>Enter a path to a folder containing images</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="liMitToFiletypeLabel">
       <property name="text">
        <string>Li&amp;mit to Filetypes:</string>
       </property>
       <property name="buddy">
        <cstring>liMitToFiletypeLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="liMitToFiletypeLineEdit">
       <property name="placeholderText">
        <string>No limitation</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="blackOutScreensLabel">
       <property name="text">
        <string>&amp;Black Out screens:</string>
       </property>
       <property name="buddy">
        <cstring>blackOutScreensCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="blackOutScreensCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QPushButton" name="formatsButton">
       <property name="text">
        <string>Formats</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QPathEdit</class>
   <extends>QWidget</extends>
   <header>qpathedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>linearModeButton</tabstop>
  <tabstop>shuffleModeButton</tabstop>
  <tabstop>randomModeButton</tabstop>
  <tabstop>loopCheckBox</tabstop>
  <tabstop>displayTimeDoubleSpinBox</tabstop>
  <tabstop>movieMinimumLoopCountSpinBox</tabstop>
  <tabstop>filterFiletypesCheckBox</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>formatsButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>ControlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>ControlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomModeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>loopCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomModeButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>loopCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
